parsers/powershell.c

```diff
@@ -55,7 +57,9 @@ typedef enum eTokenType {
 	TOKEN_SEMICOLON,
 	TOKEN_COLON,
 	TOKEN_COMMA,
-	TOKEN_KEYWORD,
+	TOKEN_KEYWORD_FUNCTION,
+	TOKEN_KEYWORD_CLASS,
+	TOKEN_KEYWORD_OTHER,
```

Comment:
Though TOKEN_KEYWORD_OTHER is defined, it is not used.
Do we need this?

Reply:
In [another comment](https://github.com/universal-ctags/ctags/pull/3476#issuecomment-1236221400), masatake suggested leaving `TOKEN_KEYWORD` as it is and adding an attribute like `token->keywordType`.

That proposal shall be addressed in a separate PR in accordance with [this comment](https://github.com/universal-ctags/ctags/pull/3476#issuecomment-1236241369).

In this PR, remove the unused TOKEN_KEYWORD_OTHER.